bims/api_views/database_record.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+from django.http import HttpResponseForbidden, HttpResponseBadRequest
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from bims.models.source_reference import DatabaseRecord
+from bims.serializers.database_record import DatabaseRecordSerializer
+
+
+class DatabaseRecordsList(APIView):
+    """API for listing all database record of source reference."""
+
+    def get(self, request, *args):
+        return Response(
+            DatabaseRecordSerializer(
+                DatabaseRecord.objects.all(), many=True).data)
+
+    def post(self, request, *args):
+        if not request.user.is_authenticated():
+            return HttpResponseForbidden()
+        data = request.data
+        try:
+            record = DatabaseRecord.objects.create(
```

Comment:
probably better to do with get_or_create, to avoid data with same fields created twice